playbook/package.json

```diff
@@ -73,21 +73,21 @@
     "@popperjs/core": "^2.6.0",
     "@rails/webpacker": "^4.3.0",
     "babel-loader": "^8.0.5",
-    "babel-plugin-flow-runtime": "0.11.1",
+    "babel-plugin-flow-runtime": "^0.11.1",
     "babel-plugin-jsx-control-statements": "^4.0.0",
-    "babel-plugin-transform-decorators-legacy": "1.3.4",
-    "babel-plugin-transform-flow-strip-types": "6.22.0",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-plugin-transform-flow-strip-types": "^6.22.0",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.23",
     "circular-dependency-plugin": "^5.2.2",
     "classnames": "^2.2.6",
-    "css-loader": "3.6.0",
-    "css-modules-flow-types-loader": "0.3.1",
-    "es5-shim": "4.5.9",
+    "css-loader": "^3.6.0",
+    "css-modules-flow-types-loader": "^0.3.1",
+    "es5-shim": "^4.5.9",
     "fast-sass-loader": "^1.5.0",
     "flatpickr": "^4.6.6",
-    "flow-bin": "0.137.0",
-    "flow-runtime": "0.14.0",
-    "highcharts": "7.2.2",
+    "flow-bin": "^0.137.0",
+    "flow-runtime": "^0.14.0",
+    "highcharts": "^7.2.2",
     "jest-axe": "^4.1.0",
     "jsx-control-statements": "^3.2.8",
     "lazysizes": "^5.2.2",
```

Comment:
Realistically, most of these should be `devDependencies` which means you shouldn't care at all about the version. The only thing we ship non-transpiled is our SCSS tokens which are pretty standard and shouldn't really care about the version of `node-sass` either.

Reply:
I'm all for moving things to devDependencies. If you can list which ones can move and which have to stay, I'm totally up for that.

Reply:
I would say everything here in `package.json` can be moved to devDependencies as the only ones we are truly concerned about are in peerDependencies currently. These being in the wrong place has been a matter of contributors not using the `-D` flag when installing a library and/or not understanding what goes where and why.